client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.css

```diff
@@ -0,0 +1,19 @@
+.opened-rma {
+  margin: 20px;
+}
+
+.rma-line {
+  margin-bottom: 10px;
+}
+
+.rma-detail-label {
+  color: #59697D;
```

Comment:
Maybe we can refer to a color that came from the ovh-ui-kit. Isn't it?

see: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/color